Nozbe/WatermelonDB
# Fix crash on android < 5.0

For some reason, this line here `delete from sqlite_master` causes database corruption in `android < 5.0` and the app would always crash on startup.

In this PR, I only delete the necessary metadata from `sqlite_master` instead of everything.

#### Test Plan 
1. Build the example RN repo post this PR 
2. Observe the app would crash with database corruption error on log cat
3. Rebuild with this PR
4. Observe everything turns out to work great.
